ethereum/contracts/BeefyClient.sol

```diff
@@ -168,7 +168,6 @@ contract BeefyClient is Ownable {
         latestBeefyBlock = _initialBeefyBlock;
         currentValidatorSet = _initialValidatorSet;
         nextValidatorSet = _nextValidatorSet;
-        renounceOwnership();
```

Comment:
Please add a code comment explaining why we are making this change, and that it is only temporary. Otherwise security auditors will likely raise concerns that the contract can be initialized multiple times.

Reply:
Comments added. Also created a ticket to add this back once full beefy is supported.

Reply:
@alistair-singh make sure tickets are tagged as security critical too plz

Reply:
I have tagged the ticket with `Audit`, `Security` and `Critical`.